### ISSUE TITLE
Please consider adding examples for Test Result Assertion with actual expected JSON expressions and response data

### ISSUE THREAD

**Issue by bigale** [StuartChartersE22/insomnia-plugin-testing#2]
Wonderful!  This is probably exactly what I need :-)

Please may I ask a favor?

I tried for a day to use the patterns that you have in the examples and never got one to work.

I think with an actual working example, I could get it to work.

I'm wondering if you would consider adding one or many example(s) with a real json response and real json strings for testing and expected. 

For example:
I have a json string that returns the string I'm looking for:
$.members[?(@.emailAddress == 'email@example.com'].userInitials
I expect it to be AJE in this case

My response can be:
$.members or $ (I guess - whatever I set it to.)

Environment (I Can't figure out the right combinations to make a test)
"field-name1": ["testing JSON path", "expected JSON path" (, "DEFAULT" / "REGEX" / "CONTAINS")],
Request
"Reference name1": [JSON1, expectedJSON1]

I tried about every combination that I could think of in these files and usually got something like (as xml from my listener):
`<json type="object">
  <test-group type="object">
    <Request_00201 type="object">
      <Members type="array"/>
    </Request_00201>
    <metrics type="object">
      <passed-assertions type="number">0</passed-assertions>
      <failed-assertions type="number">0</failed-assertions>
      <skipped-values type="number">0</skipped-values>
      <skipped-references type="number">1</skipped-references>
    </metrics>
  </test-group>
</json>`

**Comment by StuartChartersE22** [comment]
Hi bigale,
Thank you for the message, hopefully it will be what you need.

I will certainly look at setting up an example that works with real data and endpoint. I will look into the error as well to see what the problem is

**Comment by jjm17** [comment]
I too appreciate this plugin but also found the example very hard to follow. However I think a big part of the problem is I think the code slightly deviates in its behaviour from what the readme suggests. I found "the "optional third argument to the field name to specify the type of equality to use" is not optional. Although I wanted the "DEFAULT" equality-matching behaviour, I had to explicity specify it. Without it, the call to "includes" on line 57 of testing-environment/result_asserter.js fails ("Cannot read property 'includes of undefined"). This is with version 4.3.0 of the plugin. The Console output for the plugin also highlights that "setBodyText()" is deprecated and to use "setBody()" instead but that doesn't appear to be causing an issue.
